control/robust.py

```diff
@@ -356,8 +362,60 @@ def mixsyn(g,w1=None,w2=None,w3=None):
     """
     nmeas = g.outputs
     ncon = g.inputs
-    p = augw(g,w1,w2,w3)
+    p = augw(g, w1, w2, w3)
+
+    k, cl, gamma, rcond = hinfsyn(p, nmeas, ncon)
+    info = gamma, rcond
+    return k, cl, info
+
 
-    k,cl,gamma,rcond=hinfsyn(p,nmeas,ncon)
-    info = gamma,rcond
-    return k,cl,info
+def makeweight(dcgain, wc, hfgain, order=1, Ts=None):
+    """ Compute a weight transfer function, noted W
+
+    Parameters
+    ----------
+        dcgain : double positive scalar
```

Comment:
Incorrect indentation; should line up with `Parameters`.  See [numpydocs](http://numpydoc.readthedocs.io).